src/functions/uploadToRSIS/application/data-mapper/__tests__/common-mapper.spec.ts

```diff
@@ -551,6 +557,33 @@ describe('mapCommonData', () => {
       .toThrow(new MissingTestResultDataError('testResult.communicationPreferences.conductedLanguage'));
   });
 
+  it('Should reject a test result with missing mandatory data (booked staff num)', () => {
+    const missingMandatory = cloneDeep(minimalInput);
+    if (missingMandatory.testResult) {
+      delete missingMandatory.testResult.examinerBooked;
+    }
+    expect(() => mapCommonData(missingMandatory))
+      .toThrow(new MissingTestResultDataError('examinerBooked'));
+  });
+
+  it('Should reject a test result with missing mandatory data (staff num)', () => {
+    const missingMandatory = cloneDeep(minimalInput);
+    if (missingMandatory.testResult) {
+      delete missingMandatory.testResult.examinerConducted;
+    }
+    expect(() => mapCommonData(missingMandatory))
+      .toThrow(new MissingTestResultDataError('examinerConducted'));
+  });
+
+  it('Should reject a test result with missing mandatory data (keyed staff num)', () => {
+    const missingMandatory = cloneDeep(minimalInput);
+    if (missingMandatory.testResult) {
+      delete missingMandatory.testResult.examinerKeyed;
+    }
+    expect(() => mapCommonData(missingMandatory))
+      .toThrow(new MissingTestResultDataError('examinerKeyed'));
+  });
+
```

Comment:
Liking the extra tests. Could you please add/update tests to include all of the various other rekey fields?

Reply:
Added an extra test where I check all the additional rekey reason fields as well.